### ISSUE TITLE
Errors using Matlab R2022a

### ISSUE THREAD

**Issue by ufechner7** [awegroup/MegAWES#3]
I am trying to run the code on Matlab R2022a. I get the following error when running Run_OneSimulation.m:
```
Error using load
Unable to read MAT-file /home/ufechner/repos/MegAWES/Lib/Common/DE2019_params.mat. Not a binary MAT-file. Try load -ASCII to read as text.

Error in initAllSimParams_DE2019 (line 47)
load('DE2019_params.mat','DE2019','constraintOut');

Error in Get_simulation_params (line 90)
            initAllSimParams_DE2019(Kite_DOF);

Error in Run_OneSimulation (line 40)
	Get_simulation_params(windspeed, Kite_DOF);
```
Any idea?

**Comment by ufechner7** [comment]
I am one step further now, I had git lfs not correctly installed.

Now I get a different error message:
```
Warning: The model 'Dyn_6DoF_v2_0_r2019b' uses Accelerator, or Rapid Accelerator mode. This license does not support these modes. The simulation will run in Normal mode. 
> In Run_OneSimulation (line 48) 
Error using Run_OneSimulation
The block 'Dyn_6DoF_v2_0_r2019b/RequirementModelsAndLogging/Power_convergence/Delay1' performs rate transition. For robust performance, replace these blocks with actual Rate Transition blocks. For help in replacing with rate transition
block and other upgrade issues, use the Simulink Upgrade Advisor.
```

**Comment by ufechner7** [comment]
I installed Matlab 2019b, and for now it is working fine...

**Comment by rschmehl** [comment]
@ufechner7: can we close this issue? It seems like this is resolved.

**Comment by doyounyu** [comment]
> I am one step further now, I had git lfs not correctly installed.
> 
> Now I get a different error message:
> 
> ```
> Warning: The model 'Dyn_6DoF_v2_0_r2019b' uses Accelerator, or Rapid Accelerator mode. This license does not support these modes. The simulation will run in Normal mode. 
> > In Run_OneSimulation (line 48) 
> Error using Run_OneSimulation
> The block 'Dyn_6DoF_v2_0_r2019b/RequirementModelsAndLogging/Power_convergence/Delay1' performs rate transition. For robust performance, replace these blocks with actual Rate Transition blocks. For help in replacing with rate transition
> block and other upgrade issues, use the Simulink Upgrade Advisor.
> ```

This error occurs in MATLAB 2022b version too. Is there any suggestion that I could try?

**Comment by ufechner7** [comment]
Use an older version. Or ask the Matlab support how to fix it.

**Comment by doyounyu** [comment]
Yes, it works in 2019b version. 
I thought that I could find other way to use in 2022b, but I should stick with the 2019b version..
Thanks anyways!

**Comment by DylanEij** [comment]
I have fixed the issue with the delay blocks in the latest release, it turns out the newer matlab/simulink version does not allow delay blocks with inherited (-1) sample time after a rate transition block. The rate transition blocks are removed, and the Matlab function inside the subsystem of the power convergence is now executed at a specific sample time to keep the overall behaviour precisely the same as before the modifications. Therefore, I could use these modifications also in the older Simulink models without changing their behaviour in R2019B. I was able to run the simulation now in R2022B, but I expect the same for R2022A as the error was similar.
